src/main/java/com/fvv/bookstore/util/FileUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.fvv.bookstore.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * Class to save and read a file.
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 2018</p>
+ * @version 1.0 	
+ *
+ */
+public class FileUtil {
+
+	/**
+	 * To create an audit file.
+	 * 
+	 * @param message to save in the file.
+	 */
+	public static void addAuditMessage(final String message) {
+		File file = new File(Constants.FILE_PATH);
+		PrintWriter pw = null;
```

Comment:
please use the try-with-resources here to avoid the finally block

Reply:
ok